action.yml

```
@@ -29,6 +29,10 @@ inputs:
     description: "Fail if an unpinned action/image is found"
     required: false
     default: "false"
+  in_place:
+    description: "Update the files in place"
+    required: false
+    default: "false"
```

So, I agree that the existing action is a bit weird -- what would you think about making this be a directory path argument, where the default `""` value means to do the existing fetch-from-github behavior?  i.e.

```suggestion
  repo_root:
    description: "Operate on files in the specified filesystem location.  If unspecified, check out files from the current repo."
    required: false
    default: ""
```